development/process.md

```diff
@@ -82,6 +82,12 @@ keep the amount of rework from the contributor to a minimum.
     3. Go to step 5.
 
 8. Maintainer merges the pull request after final changes are accepted.
+
+    * Approval of a minimum of two maintainers is required for merge. This
```

Comment:
One maintainer approving is sufficient.

Reply:
Well.. yes.. but the total is always two .. the maintainer who merges.. and one other... that was my misunderstanding. Not sure how to word this better yet.

Reply:
Actually, no. If it’s a contribution from someone who’s not a maintainer it only requires one maintainer to review and merge

Reply:
oh .. so the exception is the other way around ... I will update tomorrow after I figure out some clear wording for that.

Reply:
There’s no exception. Any contribution needs to be reviewed by a maintainer that wasn’t the author of the contribution

Reply:
Yeah ... "that wasn't the author of the contribution" is the key here.

Reply:
I think I addressed this in the wording now.